addons/im_livechat/models/mail_message.py

```diff
@@ -29,8 +29,9 @@ def _message_format(self, fnames, format_reply=True, legacy=False):
                         'id': message_sudo.author_id.id,
                         'user_livechat_username': message_sudo.author_id.user_livechat_username,
                     }
+                # sudo: chatbot.script.step - skip ACLs when reading chatbot partner in the channel
```

Comment:
sudo comments should explain why the action is acceptable, not what it is doing. For example, you might say 'members of a channel can access the current chatbot step.'